lua/entities/gmod_wire_expression2/core/e2lib.lua

```diff
@@ -48,10 +48,10 @@ function E2Lib.setAng(ent, ang)
 end
 
 --Blacklist format:
---<top folder the material is in>[%./\\]+<material name>
+--<top folder the material is in>[%./\\]+<material name>$
 --Should prevent work-arounds like pp/./copy pp/./././copy pp\\copy etc.
 local material_blacklist = {
-	"pp[%./\\]+copy"
+	"pp[%./\\]+copy$"
 }
 local function validMaterial(material)
 	local lower = string.lower(material)
```

Comment:
Pretty sure you're wrong about that allowing 4 different materials because valve's file system works similarly to windows. Even still, windows users would not care anyway. And yes, that was added before linux plagued source, of course you would jump to the conclusion that the author is a noob though.

I can replace this with my directory normalization code, but it's unnecessary. You're just blowing everything way out of proportion.